### DISCUSSION TITLE
Collapsible Rows/ Metrics in Pivot Table or Table

### DISCUSSION THREAD

**Discussion by lucasprshepherd** [apache/superset#27555] [Q&A / Help]
Hello world,

I am currently working on a project that calls for a collapsible roll-up table. I have found some alternative approaches, but I am curious if anyone has proposed or created such a feature. Ideally, I am looking to have a pivot table with some metric that can be clicked to reveal other sub-metrics related to this metric. 

Similar functionality is available in Excel, for reference: [https://www.youtube.com/watch?v=mUOp0WU4ysE](https://www.youtube.com/watch?v=mUOp0WU4ysE)

Any feedback would be highly appreciated! 😁

**Comment by joshstackintegrated** [comment]
+1 collapsible pivot table i feel is an important chart to have

**Reply by joshstackintegrated** [reply]
I feel silly, but there is a way to collapse it. By enabling the show row subtotals
![image](https://github.com/apache/superset/assets/149730150/b365ced5-3ec0-4ad6-a011-acc907ea46a5)

**Reply by lucasprshepherd** [reply]
I stumbled across that feature as well - pretty neat. What I was looking for was dropdown functionality without the need for aggregation or at least some more flexibility in aggregation. Like suppose you have a group of metrics that are all related - I want the most important (for business purposes) metric to unroll and show the related but less important metrics.

**Comment by moksh-enf** [comment]
Is there any way to have it collapsed by default?

**Reply by jonaschn** [reply]
Same question is still not answered on [StackOverflow](https://stackoverflow.com/questions/79234297/collapsed-rows-or-columns-in-pivot-table-by-default)

**Comment by pellepelle3** [comment]
The reason I feel like this is valuable is is i believe it would improve the performance of large pivot tables as it doesn't need to render if expanded.
